sdk/purview/azure-purview-scanning/tests/test_smoke.py

```diff
@@ -5,17 +5,13 @@
 # license information.
 # -------------------------------------------------------------------------
 from testcase import PurviewScanningTest, PurviewScanningPowerShellPreparer
-from azure.purview.scanning.rest import data_sources
+
 
 class PurviewScanningSmokeTest(PurviewScanningTest):
 
     @PurviewScanningPowerShellPreparer()
     def test_basic_smoke_test(self, purviewscanning_endpoint):
         client = self.create_client(endpoint=purviewscanning_endpoint)
-        request = data_sources.build_list_all_request()
-        response = client.send_request(request)
-        response.raise_for_status()
-        assert response.status_code == 200
-        json_response = response.json()
-        assert set(json_response.keys()) == set(['value', 'count'])
-        assert len(json_response['value']) == json_response['count']
+        response = client.data_sources.list_all()
+        result = [item for item in response]
+        assert result == []
```

Comment:
it's kind of weird that the pages are returning nothing, can you fix the tests by either adding values to the purview account so we have actual return values here? thanks!